sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathFunctionsSuite.scala

```diff
@@ -191,6 +192,24 @@ class MathFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     testUnary(Log2, f, (-5 to -1).map(_ * 1.0), expectNull = true)
   }
 
+  test("sqrt") {
+    testUnary(Sqrt, math.sqrt, (0 to 20).map(_ * 0.1))
+    testUnary(Sqrt, math.sqrt, (-5 to -1).map(_ * 1.0), expectNull = true)
+
+    val inputSequence = Seq(4.5, 10.10, 20.55, 36.0, 57.0, 61.7, 77.0, 81.0, 98.9)
```

Comment:
actually - question: why having this when you already have the testUnary call up there?